### PR TITLE
CDS documentation

### PR DESCRIPTION
**Pull request by reshmakh**

### CODE REVIEW

**Comment by reshmakh** [comment]
Documentation related to #3003

File: packages/docs/docs/careplans/clinical-decision-support.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Medplum is not currently certified for (a)(9) but is pursuing certification. Subscribe to [github issue](https://github.com/medplum/medplum/issues/3003) for updates.
+'''
+
+## Predictive
+
+Predictive clinical decision support technology is "intended to support decision-making based on algorithms or models that derive relationships from training or example data and then are used to produce an output or outputs related to, but not limited to, prediction, classification, recommendation, evaluation, or analysis.”
+
+Large language model based clinical decision support tools, as well as tools that use algorithms for risk assessment or triage fall in the predictive clinical decision support category.
+
+Medplum enables [many implementations](/case-studies) with predictive clinical decision support. Per the HTI final ruling, the predictive clinical decision support systems will become regulated in Dec. 2024. Any system certified to g10, b2, f1 or e1 will be required to provide **Insight Reports** as part of maintaining their certification.
```
Comment (codyebberson) [inline packages/docs/docs/careplans/clinical-decision-support.md:23]: "Dec." abbreviation is weird here.  Recommend spelling out "December"

File: packages/docs/docs/careplans/clinical-decision-support.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+### Insights Reporting
+
+Electronic health records that support predictive clinical decision support will be required to report on the usage of their product. Prepare the following basic statistics as part of certification: the number of times the decision support was used, how many unique clinicians used it, the number of times it was updated and any complaints received.
+
+## Linked Referential
+
+Linked referential clinical decision support are hyperlinks that link to reference material that is specific to the clinical context of a specific patient or population. The ONC criteria that define this standard are 170.205(a)(3,4) amd relate to the retrieving context-aware knowledge using the HL7 Infobutton.
+
+'''caution
```
Comment (codyebberson) [inline packages/docs/docs/careplans/clinical-decision-support.md:43]: Use backticks, not normal quotes.

File: packages/docs/docs/careplans/clinical-decision-support.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Medplum is not certified for (a)(3,4) but serves as a basis for those who wish to implement. The Clinical Profile and [Diagnostic Report](https://storybook.medplum.com/?path=/story/medplum-diagnosticreportdisplay--simple) react components serve as common launch points for Infobutton implementations.
+'''
+
+To support the linked referential clinical decision support the system should be capable of retrieving information based on one or more of the following data elements. The Clinical Profile react component highlights the data elements.
+
+TODO: Link to the [Clinical Profile](https://github.com/medplum/medplum/issues/3625) react component
```
Comment (codyebberson) [inline packages/docs/docs/careplans/clinical-decision-support.md:49]: Let's remove this, but add a comment to issue #3625 to update this doc when the component is finished?

File: packages/docs/docs/careplans/clinical-decision-support.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+- Medications
+- Smoking status
+
+[UpToDate](https://www.wolterskluwer.com/en/solutions/uptodate/uptodate-advanced/workflow-integration) is a common provider for linked referential data.
+
+## Evidence Based
+
+Evidence-based clinical decision support systems are largely related to medication administration, drug interactions (with other drugs, foods, OTC medications, etc.) and dosing. Evidence-based clinical decision support systems plug into health record systems via SMART-App-Launch links, iFrames or APIs, all of which are supported by Medplum.
+
+TODO: Link to the the [Smart-App-Launch react component](https://github.com/medplum/medplum/issues/3605)
```
Comment (codyebberson) [inline packages/docs/docs/careplans/clinical-decision-support.md:62]: Let's remove this, but add a comment to issue #3605 to update this doc when the component is finished?

**Review by codyebberson — APPROVED** [review]
Nice, ty 🙏  Content looks good 👍 

Minor note on the admonitions blocks... I'm not sure it's doing what you wanted it to do.

See preview: https://medplum-www-git-reshma-cds-docs-medplum.vercel.app/docs/careplans/clinical-decision-support

image

I suspect what you really wanted is Docusuarus Admonitions: https://docusaurus.io/docs/markdown-features/admonitions

image

In which case, the syntax would be colons and `warning` (`caution` isn't supported):

```
:::warning

Some **content** with _Markdown_ `syntax`. Check [this `api`](#).

:::
```

**Comment by reshmakh** [comment]
@codyebberson - yes appreciate that pointer.  I did want the admonitions.  I'm also at a loss on how to get this item to show up in the sidebar under /docs/careplans - resubmitting admonitions now.

**Pull request merged**